lib/fastlane/plugin/wpmreleasetoolkit/helper/github_helper.rb

```diff
@@ -7,8 +7,19 @@ module Fastlane
 
   module Helper
     class GithubHelper
+      def self.github_token
+        token = [
+          'GHHELPER_ACCESS', # For historical reasons / backward compatibility
+          'GITHUB_TOKEN',    # Used by the `gh` CLI tool
+        ].map { |key| ENV[key] }
+                .compact
+                .first
+
+        token || UI.user_error!('Please provide a GitHub authentication token via the `GITHUB_TOKEN` environment variable')
+      end
+
       def self.github_client
-        client = Octokit::Client.new(access_token: ENV['GHHELPER_ACCESS'])
+        client = Octokit::Client.new(access_token: github_token)
```

Comment:
Assuming you apply the previous suggestion on line 10, then:
```suggestion
        client = Octokit::Client.new(access_token: github_token!)
```